### ISSUE TITLE
Latest merged PR is not being applied in NPM

### ISSUE THREAD

**Issue by davidlewallen** [ctimmerm/axios-mock-adapter#117]
Just updated my axios-mock-adapter dependency and I am still unable to overwrite my mocks. Does the npm version need to be updated?

**Comment by ctimmerm** [comment]
Yeah, it's because a new version hasn't been published yet.

**Comment by DylanVann** [comment]
The npm versions of this package do not match the versions on the release page.

**Comment by ctimmerm** [comment]
@DylanVann can you elaborate?

**Comment by DylanVann** [comment]
Great package btw, I've found it useful.

So the version on npm right now that was released 18 hours ago is `1.15.0`.

<img width="1099" alt="screen shot 2018-04-06 at 11 29 45" src="https://user-images.githubusercontent.com/1537615/38429929-db7265e6-398d-11e8-9b3c-8e8f62e62f63.png">

But on the releases page on github:

<img width="808" alt="screen shot 2018-04-06 at 11 31 31" src="https://user-images.githubusercontent.com/1537615/38430021-162af810-398e-11e8-8879-5ec1a11e0898.png">

Actually my mistake, `1.15.0` is higher than `1.8.1`. I didn't see all the collapsed tags. There just aren't release notes.
